web/src/_variables.scss

```diff
@@ -13,12 +13,14 @@ $colors-border-hover: var(--calculated_secondary-border-1);
 $colors-main-border: var(--calculated_important-border);
 $footer-border-color: var(--calculated_footer_border);
 $colors-super-pale-black: var(--calculated_super-pale_label-text-graphics);
+$colors-blue-link: var(--blue_link_dark);
 
 // common colors for deactivate and disabled look
 $colors-deactivate: var(--labels_secondary-inactive-label-text-graphics);
 $colors-deactivate-color1: var(--labels_inactive-button);
 $colors-deactivate-color2: var(--labels_secondary-inactive-label-text-graphics);
 $colors-deactivate-color4: var(--labels_inactive-button);
+$gray-text-color: var(--gray_text_content);
```

Comment:
If the already defined blue color is not useful here and we want to define a new color anyway, it should also be added to the color utils and the dark and light color js files.

utils/color.js
config/colors/light.js
config/colors/dark.js

Reply:
I've removed my declaration of the text color and used the previously declared text color.